uplink/storage/segments/store_test.go

```diff
@@ -223,56 +29,61 @@ func TestCalcNeededNodes(t *testing.T) {
 	} {
 		tag := fmt.Sprintf("#%d. %+v", i, tt)
 
-		rs := pb.RedundancyScheme{
-			MinReq:           tt.k,
-			RepairThreshold:  tt.m,
-			SuccessThreshold: tt.o,
-			Total:            tt.n,
+		rs := storj.RedundancyScheme{
+			RequiredShares: tt.k,
+			RepairShares:   tt.m,
+			OptimalShares:  tt.o,
+			TotalShares:    tt.n,
 		}
 
-		assert.Equal(t, tt.needed, segments.CalcNeededNodes(&rs), tag)
+		assert.Equal(t, tt.needed, segments.CalcNeededNodes(rs), tag)
 	}
 }
 
-func runTest(t *testing.T, test func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, segmentStore segments.Store)) {
-	testplanet.Run(t, testplanet.Config{
-		SatelliteCount: 1, StorageNodeCount: 4, UplinkCount: 1,
-	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
-		// TODO move apikey creation to testplanet
-		project, err := planet.Satellites[0].DB.Console().Projects().Insert(context.Background(), &console.Project{
-			Name: "testProject",
-		})
-		require.NoError(t, err)
-
-		apiKey, err := macaroon.NewAPIKey([]byte("testSecret"))
-		require.NoError(t, err)
-
-		apiKeyInfo := console.APIKeyInfo{
-			ProjectID: project.ID,
-			Name:      "testKey",
-			Secret:    []byte("testSecret"),
-		}
-
-		// add api key to db
-		_, err = planet.Satellites[0].DB.Console().APIKeys().Create(context.Background(), apiKey.Head(), apiKeyInfo)
-		require.NoError(t, err)
-
-		TestAPIKey := apiKey.Serialize()
-
-		metainfo, err := planet.Uplinks[0].DialMetainfo(context.Background(), planet.Satellites[0], TestAPIKey)
-		require.NoError(t, err)
-		defer ctx.Check(metainfo.Close)
-
-		ec := ecclient.NewClient(planet.Uplinks[0].Log.Named("ecclient"), planet.Uplinks[0].Transport, 0)
-		fc, err := infectious.NewFEC(2, 4)
-		require.NoError(t, err)
-
-		rs, err := eestream.NewRedundancyStrategy(eestream.NewRSScheme(fc, 1*memory.KiB.Int()), 0, 0)
-		require.NoError(t, err)
-
-		segmentStore := segments.NewSegmentStore(metainfo, ec, rs, 4*memory.KiB.Int(), 8*memory.MiB.Int64())
-		assert.NotNil(t, segmentStore)
-
-		test(t, ctx, planet, segmentStore)
-	})
-}
+// func runTest(t *testing.T, test func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, streamStore streams.Store)) {
```

Comment:
Do we plan to restore these in a short time?

Reply:
tests from `uplink/storage/segments/store_test.go` were moved and adjusted to `uplink/storage/segments/store_test.go`

Reply:
I need to delete this